tasks/places.rb

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+require './models.rb'
+
+namespace :places do
+  task :create do
+    places = YAML::load(File.open('places.yml'))
```

Comment:
Do not use `::` for method calls.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.